lib/restbase.js

```diff
@@ -271,7 +274,8 @@ RESTBase.prototype._request = function(req) {
         var childRESTBase = this.makeChild(req);
         childReq.params = match.params;
 
-        var accessRestricted = match.permissions
+        var accessRestricted = !this._isSysRequest(req)
```

Comment:
It might be worth adding a comment here to make it clear that sys requests are in fact still access restricted, but the check happens elsewhere.